process-invoices/print_reports_by_status.py

```diff
@@ -15,225 +15,266 @@
 import configparser
 import invoice_sap
 
+sys.path.append("..")
+from llama.alma import Alma_API_Client
+import llama.config as config
 
-config = configparser.ConfigParser()
-config.read('invoice.config')
-alma = config['ALMA']
-
+# Create some date variables for later
 today = date.today().strftime("%m/%d/%Y")
 today2 = date.today().strftime("%Y%m%d")
 rightnow = datetime.now().strftime("%Y%m%d%H%M%S")
 
-apikey = alma['apikey']
-host = alma['apihost']
-
+# Open country code config file and save in memory as a dict
 file = open("countries.txt", "r")
-
 contents = file.read()
 countries = ast.literal_eval(contents)
-
 file.close()
 
-mfile = "review_mono_report_" + rightnow + ".txt"
-sfile = "review_serial_report_" + rightnow + ".txt"
+# Create output filenames for monograph and serials review reports and open them both
+# in write mode
```

Comment:
The directory needs to exist before this point.  We either need to `https://docs.python.org/3/library/os.html#os.makedirs` or just create and commit the file in git.  Probably creating the dir in python is better so that we dont see an error if it somehow gets deleted.